src/components/Textfields/TextFieldBase/__test__/TextfieldBase.test.tsx

```diff
@@ -0,0 +1,53 @@
+import TextfieldBase, { TexfieldBaseProps } from "../";
+
+import { render, RenderResult, screen } from "@testing-library/react";
+
+describe("<TextfieldBase />", () => {
+    let wrapper: RenderResult;
+
+    const propsForSecondaryColor: TexfieldBaseProps = {
+        label: "Test",
+        value: "Value",
+        color: "secondary",
+    };
+
+    const propsForPrimaryColor: TexfieldBaseProps = {
+        label: "Test",
+        value: "Value",
+        color: "primary",
+    };
+
+    const propsWithErrorMessage: TexfieldBaseProps = {
+        label: "Test",
+        value: "Value",
+        errorMessage: "error",
+        color: "secondary",
+    };
+
+    const propsWithNoErrorMessage: TexfieldBaseProps = {
+        label: "Test",
+        value: "Value",
+        errorMessage: "",
+        color: "secondary",
+    };
+
+    it("should match snapshot", () => {
+        wrapper = render(<TextfieldBase {...propsForSecondaryColor} />);
+        expect(wrapper.container).toMatchSnapshot();
+    });
+
+    it("should match snapshot", () => {
+        wrapper = render(<TextfieldBase {...propsForPrimaryColor} />);
+        expect(wrapper.container).toMatchSnapshot();
+    });
+
+    it("should match snapshot", () => {
+        wrapper = render(<TextfieldBase {...propsWithErrorMessage} />);
+        expect(wrapper.container).toMatchSnapshot();
+    });
+
+    it("should match snapshot", () => {
+        wrapper = render(<TextfieldBase {...propsWithNoErrorMessage} />);
+        expect(wrapper.container).toMatchSnapshot();
+    });
+});
```

Comment:
Please help me edit the name of the test to be more descriptive

Reply:
I done it with one unit test for one props, you can see when review code.